compiler/optimizer/Inliner.cpp

```diff
@@ -346,6 +346,14 @@ TR_InlinerBase::setInlineThresholds(TR::ResolvedMethodSymbol *callerSymbol)
    if (e)
       _nodeCountThreshold = atoi(e);
 
+   if (comp()->getOption(TR_EnableOSR))
+      {
+      static const char *f = feGetEnv("TR_OSRNodeCountThreshold");
+      if (f)
+         _nodeCountThreshold = atoi(f);
+      else
+         _nodeCountThreshold *= 2;
```

Comment:
Note this is not trees - this is nodes. And empirical evidence so far says this number is correct - we have to generate the OSR catch block, code block, and dead stores so the number can be quite large - especially for small methods.